### PR TITLE
[RDAPI-28086] Important changes and Fixed issues updates in “20220830_apimgr_relnotes”

### PR DESCRIPTION
**Pull request by moscaribera**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by moscaribera** [comment]
Deploy preview:
https://deploy-preview-2397--axway-open-docs.netlify.app/docs/apim_relnotes/20220830_apimgr_relnotes/#organization-administrator-self-service
https://deploy-preview-2397--axway-open-docs.netlify.app/docs/apim_relnotes/20220830_apimgr_relnotes/#fixed-security-vulnerabilities

File: content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ During the Policy Studio and Configuration Studio update process, the `plugins`
 
 For more information, see [Install a Policy Studio update](/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion/#install-a-policy-studio-update) and [Install a Configuration Studio update](/docs/apim_installation/apigw_upgrade/upgrade_steps_oneversion/#install-a-configuration-studio-update).
 
+### Organization administrator self-service
+
+When self-service is enabled Organization administrator users can access APIs for the organizations they are a member of or have been granted access. Previously, the API Manager incorrectly allowed access to APIs in other organizations even if no API access was granted when self-service was enabled. Customer scripts or client applications may now fail to get other organizations APIs if the Organization administrator users have not been granted access.
```
Comment (acotter90) [inline content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md:43]: Change "Organization administrator users" to "Organization administrators".
Change "have been granted access." to "have been granted access to.".
Change "get other organizations APIs" to " get other organization's APIs"
Change second instance of "Organization administrator users" to "Organization administrators".
Change last sentence ending from "have not been granted access." to "have not been granted access to these APIs."

I.e. I would go with something like

When self-service is enabled Organization administrators can access APIs for the organizations they are a member of or have been granted access to. Previously, the API Manager incorrectly allowed access to APIs in other organizations even if no API access was granted when self-service was enabled. Customer scripts or client applications may now fail to get other organization's APIs if the Organization administrators have not been granted access to these APIs.

File: content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md [hunk 2]
```diff
@@ -66,7 +72,9 @@ This version of API Gateway and API Manager includes:
...
+| RDAPI-27290 | 01365642 |                | **Issue**: When `api.manager.orgadmin.selfservice.enabled` system property is set to `true` an organization administrator can see APIs of other organizations. **Resolution**: When self-service is enabled an organization administrator can only see APIs for organizations he/she is a member of or has been granted access. |
```
Comment (acotter90) [inline content/en/docs/apim_relnotes/20220830_apimgr_relnotes.md:77]: Change "has been granted access." to "has been granted access to."

**Comment by acotter90** [comment]
Looks good to me! Cheers!

**Comment by andreamussap** [comment]
> Looks good to me! Cheers!

Thanks, @acotter90 for this review!

**Pull request merged**